kernel/kselftest.py

```diff
@@ -54,7 +54,7 @@ def setUp(self):
                          'libpopt-dev', 'libcap-ng0', 'libcap-ng-dev',
                          'libnuma-dev', 'libfuse-dev', 'elfutils', 'libelf1'])
         elif 'SuSE' in detected_distro.name:
-            deps.extend(['popt', 'glibc', 'glibc-devel', 'popt-devel',
+            deps.extend(['popt', 'glibc', 'glibc-devel', 'popt-devel', 'sudo',
```

Comment:
why 'sudo' rpm is required i don't think we are running this test on suse on non root user can you please help me here

Reply:
Yes, that is right. There are tests which by default runs with `sudo` command and hence failing just because command is not found.

Reply:
sure